pisek/visualize.py

```diff
@@ -135,18 +155,23 @@ def _evaluate_results(
     def from_log(
         name: str, config: TaskConfig, testing_log, limit: float
     ) -> "SolutionResults":
-        if name not in testing_log:
+        if name not in testing_log["solutions"]:
             raise MissingSolution(name)
 
         results = []
-        for result in testing_log[name]["results"]:
+        for test_name, result in testing_log["solutions"][name]["results"].items():
+
+            def tryDecimal(s: Optional[str]) -> Optional[Decimal]:
```

Comment:
This method should have a `snake_case` name.
I also don't think `try` is the proper word here, something like `optional` or `maybe` would be more appropriate.